source/install/install-ubuntu-2204-server.rst

```diff
@@ -0,0 +1,18 @@
+.. _install-ubuntu-2204-server:
+
+Installing Ubuntu Server 22.04 LTS
+----------------------------------
+
+Install the 64-bit version of Ubuntu Server on each machine that hosts one or more of the components.
+
+**To install Ubuntu Server 22.04:**
+
+1. To install Ubuntu Server 22.04, see the `Ubuntu Installation Guide. <https://help.ubuntu.com/22.04/installation-guide/amd64/index.html>`__
```

Comment:
Docs aren't published by Ubuntuy yet, so need to get this link validated once they are.

Reply:
Just reviewing this again, it's been moved per the text on https://help.ubuntu.com/

> As of the Ubuntu LTS release in 2020, the server documentation has moved to[ a different site](https://ubuntu.com/server/docs), and will automatically update when changes are made to the [discourse source code pages](https://discourse.ubuntu.com/t/introduction/11322).

So, it looks like https://ubuntu.com/server/docs/installation might be the best link. I can update it with this if we'd want. I know we are moving away from OS specific instructions a bit too, so i can close this PR down if needed.